server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/modules/blobstore/BlobStoreChoosingModule.java

```diff
@@ -82,10 +82,10 @@ BlobStore provideBlobStore(BlobStoreChoosingConfiguration choosingConfiguration,
                 return swiftBlobStoreProvider.get();
             case CASSANDRA:
                 return cassandraBlobStoreProvider.get();
-            case UNION:
```

Comment:
optional: you can keep UNION and mark it as Deprecated, then in switch case block, you can throw out a decent error saying UNION is no longer supported, that is more user friendly IMO

Reply:
> optional: you can keep UNION and mark it as Deprecated, then in switch case block, you can throw out a decent error saying UNION is no longer supported, that is more user friendly IMO

We agreed in the ADR to replace it.

Note that hybrid blobStore allows reading the content of both object stores and thus allows retro-compatibility...